src/test/java/org/opentripplanner/graph_builder/module/osm/TestOpenStreetMapGraphBuilder.java

```diff
@@ -39,6 +39,7 @@
     @Before
     public void setUp() {
         extra = new HashMap<>();
+        Locale.setDefault(new Locale("en", "US"));
```

Comment:
I would like to se a tearDown method where you set the default locale back. The reason is that if another test have the same issue as this on it might run ok/not ok depending on running before or after this test. Vi do not want to set the locale for every test - witch also solve it.

Reply:
As general functionality I think server software that depends on system locale is a bit oddity. It would be preferred to choose locale settings per served request using e.g. `Accept-Language` HTTP header. However docs/Configuration.md do contain documented option to set default locale, which I believe override system locale (which is probably LANG=C in most instances anyway).

So I was thinking also alternatively to just remove this one line from unit test that depends on system locale to avoid changing the locale settings https://github.com/opentripplanner/OpenTripPlanner/blob/5b55cdf2298dd1f20313797030a7366d8ae43aa4/src/test/java/org/opentripplanner/graph_builder/module/osm/TestOpenStreetMapGraphBuilder.java#L308

But adding `tearDown` is of course a correct way to handle this in case there would be more unit tests depending on system lcoale.

Reply:
I totally agree that we should cleanup the TimeZone  issues in OTP, but it is a bigger task and I believe there is an issue for it. But, until then all tests that depend on the system timezone (which we should not do) should use the setup and tearDown.

How to handle TimeZone in OTP is a big discussion involving CPU prefetching(time-table compression optimization witch prevent us from using epoch) , multiple data-sources with relative board/alight times and API specific use-cases (user TX, stop/loc TZ, agency TZ - which all are valid depending on the UC).

Reply:
I went now with easiest solution by removing that assertion. Also force-pushed and rebased on top of current dev-2.x. It was actually a bit unclear for me if rebase or merge commits are preferred when reading docs/Developer-Guide.